src/Kunstmaan/AdminBundle/Command/CreateUserCommand.php

```diff
@@ -100,20 +113,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $user = $em->getRepository($userClassName)->findOneBy(array('username' => $username));
 
         // Attach groups
-        $groupNames = explode(',', $groupOption);
         /** @var Group[] $groups */
-        $groups = $this->getContainer()->get('fos_user.group_manager')->findGroups();
         $groupOutput = '';
 
         foreach ($groupNames as $groupName) {
-
-            if ((int)$groupName !== 0) {
-                $group = $em->getRepository('KunstmaanAdminBundle:Group')->findOneBy(array('name' => $groups[$groupName]->getName()));
-                $groupOutput .= $groups[$groupName]->getName() . ', ';
-            } else {
-                $group = $em->getRepository('KunstmaanAdminBundle:Group')->findOneBy(array('name' => $groupName));
-                $groupOutput .= $groupName . ', ';
-            }
+            $group = $em->getRepository('KunstmaanAdminBundle:Group')->findOneBy(array('name' => $groups[$groupName]->getName()));
```

Comment:
Why is the group loaded again here? We already loaded them above.

Reply:
That's a very solid point, I'll fix that one too.